### PR TITLE
Add updates for dragster book

### PR DESCRIPTION
**Pull request by yurybotov**
Add updates for dragster book: additional constructor and constants

### CODE REVIEW

File: keywords.txt [hunk 1]
```diff
@@ -28,4 +28,7 @@ led 			KEYWORD2
 
 SWAP_LEFT		LITERAL1
 SWAP_RIGHT		LITERAL1
-SWAP_BOTH		LITERAL1
\ No newline at end of file
+SWAP_BOTH		LITERAL1
+MMAX_3_OHM     LITERAL1
+MMAX_16_OHM        LITERAL1
+PWM_OF_START_MOVING     LITERAL1
```
Comment (nkrkv) [inline keywords.txt]: Tab/spaces inconsistency. Use either, not both

**Review by nkrkv — CHANGES_REQUESTED** [review]
Looks fine.

I suggest using a single term though. Either “Upper Limit” or “Maximum”. Mixing both causes extra cognitive load. I suggest using Min and Max everywhere because they are shorter:

- `MMAX_3_OHM`
- `motorMax`
- `motorMinBackward` / `motorMinForward`
- `MMIN_DEFAULT` (current `PWM_OF_START_MOVING`)

File: keywords.txt (hunk 1 above)
Comment (yurybotov) [reply keywords.txt]: Yes, auto-replacer worked. Switch-off.

**Review by yurybotov — COMMENTED** [review]
done

**Review by Matthew3000 — APPROVED** [review]
thumbs up

**Review by nkrkv — COMMENTED** [review]
Ctrl+F on “Limit” please

**Comment by yurybotov** [comment]
Local variables? I don`t want rename setMotorLimits.

**Review by nkrkv — APPROVED** [review]
OK, let it be. Indeed, out of this PR scope. LGTM

Try to always keep a single term for a single concept to make the code easier to understand.

**Pull request merged**